### PR TITLE
Golden-View-School

### PR DESCRIPTION
**Pull request by ernestkorir**
>  Build the capstone while containing the Main page and the about page

> It has two versions of display ie mobile version up to a screen of 768px and above it switches to the Desktop version

> Link the Logo to the home or Main Page

> The hamburger button only appears in the mobile version and provides a Menu-list

> The mobile menu closes when you click on the (X) button 

> The section "Featured speakers"  is created dynamically through js

Thanks for reviewing my PR

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  </div>
+  <nav id="nav" class="invisible">
+    <div id="cross" class="cross-container" onclick="toggleMenu()">
+      <i class="fas fa-times"></i>
+    </div>
+    <a href="./index.html" class="nav-logo-frame">
+      <img src="./assets/Golden-view.jpg" alt="" class="nav-logo">
+    </a>
+    <ul class="menu">
+      <li><a href="./about.html" onclick="toggleMenu()">About</a></li>
```
Comment (topeogunleye) [inline about.html]: - [ ] It is redundant to have an `àbout` link on the about page. I kindly suggest that you replace this about page link with the home page link.

## EXAMPLE
` <li><a href="./index.html" onclick="toggleMenu()">Home</a></li>`


image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+      <h3>2022.10.17(Monday) ~ 22(SAT)</h3>
+      <p>
+        @ Highway Towers Building , Nakuru Kenya
+      </p>
+      <img src="./assets/hitmark-gray.png" alt="" class="tag-hitmark">
+      <i class="fas fa-globe-americas"></i>
+    </div>
+  </header>
+  <main>
+    <h2>Main Program</h2>
```
Comment (topeogunleye) [inline index.html:74]: - [ ] I kindly suggest that you make sure that all the program articles are of the same size in width and length like it is in the Behance Design

|     |   |
| ---      | ---       |
| Behance Design | image      |
| Your Design   | image  |

- [ ] [OPTIONAL] I kindly suggest that you add the white border on hover.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+  <div class="pre-nav">
+    <a href=""> <i class="fab fa-twitter"></i></a>
+    <a href=""><i class="fab fa-instagram"></i></a>
+    <ul class="pre-menu">
+      <li><a href="#">English</a></li>
+      <li><a href="#">My Page</a></li>
+      <li><a href="#">Logout</a></li>
+    </ul>
+  </div>
+  <nav id="nav" class="invisible">
```
Comment (topeogunleye) [inline index.html:26]: - [ ] I kindly suggest that you ensure that the background of your navbar is transparent or the same color/pattern as the top section background

image

File: README.md [hunk 4]
```diff
@@ -1,56 +1,39 @@
...
+## Live Demo
 
+[Live Demo](https://ernestkorir.github.io/Capstone-1/)
 
-## Getting Started
+## Loom Video
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Loom Video](https://www.loom.com/share/7225df2afc564e04aaebc5995effe2ed)
```
Comment (topeogunleye) [inline README.md]: - [ ] Please remember to highlight a piece of the code the you find interesting or proud of making

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @ernestkorir, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
## (Highlights) Good Points: 
* Good commit messages. :heavy_check_mark:
* No linter errors. :heavy_check_mark:
* Followed Github flow :heavy_check_mark:
* Professional README file. :heavy_check_mark:

_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Please check my inline comments.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+    <div class="img-curtain">
+      <img src="./assets/11-chess-board-gray-resized.png" alt="" class="curtain-deco">
+      <img src="./assets/hitmark-gray.png" alt="" class="curtain-hitmark">
+    </div>
+  </div>
+  <header>
+    <h1>Hey There</h1>
+    <h2>Golden View <br>SChool<br>Welcomes you</h2>
+    <div class="deco-ball-1"></div>
+    <div class="txt-container">
```
Comment (ghost) [inline index.html:58]: - [ OPTIONAL] Kindly try to give a margin to your elements correctly to match the original design 

image

### Checkout the original design

image

- [OPTIONAL] Kindly provide a margin top and bottom of this section to match the original design

image

### Checkout the original design

image

- [OPTIONAL] Kindly enable this hover effect on your cards (white borderline)  to match the original design

image

**Review by frank1738 — APPROVED** [review]
Hi @ernestkorir 

Good work implementing the changes requested. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

YesYouAreGIF (2)


## Highlights

✅ Correct use of Github workflow
✅ No linter error
✅ README is okay

## Optional suggestions
- [ ] There are some optional comments I have highlighted below but I would highly recommend that you implement them to improve your website.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**